www/src/components/clusters/Clusters.tsx

```diff
@@ -1,135 +1,354 @@
 import { useQuery } from '@apollo/client'
 import { Box } from 'grommet'
 import {
-  A, Br, Flex, Span,
+  Button,
+  Flex,
+  MenuItem,
+  P,
+  Select,
+  Span,
 } from 'honorable'
-import { Button, EmptyState, LoopingLogo } from 'pluralsh-design-system'
-import { ReactElement, useEffect, useState } from 'react'
-import { Link } from 'react-router-dom'
+import moment from 'moment'
+import {
+  Chip,
+  ClusterIcon,
+  ReloadIcon,
+  StatusIpIcon,
+  StatusOkIcon,
+} from 'pluralsh-design-system'
+import { useEffect, useMemo, useState } from 'react'
+
```

Comment:
It looks like `Clusters.tsx` got overwritten by a converted `Clusters.js`, which I'm guessing isn't what we want, right?

Reply:
Rebased